main.py

```diff
@@ -0,0 +1,81 @@
+from typing import Type, Optional
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
```

Comment:
U need to automatically create db tables before starting the project
https://github.com/mate-academy/py-fastapi-library-management-api#:~:text=Create%20the%20database%20tables%20using%20the%20SQLAlchemy%20models.
Otherwise it will throw an error
![image](https://user-images.githubusercontent.com/86322722/231730113-3da3f3ee-6359-4b68-8d20-56adf1cc59d3.png)